cmd/server-manager/plugins/events.lua

```diff
@@ -12,20 +16,49 @@ function onEventStart(encodedRaceConfig, encodedServerOpts, encodedEntryList)
     local serverOpts = json.decode(encodedServerOpts)
     local entryList = json.decode(encodedEntryList)
 
+
     -- Uncomment these lines and run the function (start any event) to print out the structure of each object.
     --print("Race Config:", utils.dump(raceConfig))
     --print("Server Options:", utils.dump(serverOpts))
     --print("Entry List:", utils.dump(entryList))
-
+    --print("Countrycodes", utils.dump(countryCodes))
     -- Function block NOTE: this hook BLOCKS, make sure your functions don't loop forever!
-    -- in order to use the weatherAPI you need to get a free API key from https://openweathermap.org/
-    -- uncomment functions to enable them!
-    --raceConfig, serverOpts = weatherAPI(raceConfig, serverOpts, "get-an-api-key-from-https://openweathermap.org/")
+    --Uncomment this line to set Weather API On, don't forget to put you openweathermap API key line 55
+    getTrackInfo(raceConfig, serverOpts)
```

Comment:
This might be me not knowing how lua works with pointers, but does this function call not need to assign the returned values:

`raceConfig, serverOpts = getTrackInfo(raceConfig, serverOpts)`

If it doesn't then TIL, but it still might be more readable for new users :)

Reply:
Also here I think I'd rather have this main function act more as a hub for calling functions:
```
raceConfig, serverOpts = getTrackInfo(raceConfig, serverOpts)

raceConfig, serverOpts = weatherAPI(raceConfig, serverOpts, "get-an-api-key-from-https://openweathermap.org/")
```
That way it's a little easier to follow the flow, rather creating a chain of functions.